matrix-sdk/src/main/java/org/matrix/androidsdk/fragments/MatrixMessageListFragment.java

```diff
@@ -737,6 +737,7 @@ public void run() {
                 mMessageListView.setSelection(mAdapter.getCount() - 1);
             }
         }, Math.max(delayMs, 0));
+        mMessageListView.smoothScrollBy(0, 0);
```

Comment:
Ok, I've checked and it works, thanks. Maybe move this new line above `mMessageListView.postDelayed(new Runnable() {` and add a comment, for instance: `// Stop the current scrolling of the list if any`

Reply:
I actually moved it inside the postDelayed method. If there was a long delay and it was outside the method the adapter would stop, wait a bit, then skip to the bottom. That might be confusing. Should be fixed now though.